plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/ConvertJobSuspender.java

```diff
@@ -61,7 +61,7 @@ private SuspendedJob(Job job, long scheduleDelay) {
   static Predicate<Job> isConvertJob = new Predicate<Job>() {
     @Override
     public boolean apply(Job job) {
-      return CONVERT_JOB_CLASS_NAME.equals(job.getClass().getName());
+      return job != null && CONVERT_JOB_CLASS_NAME.equals(job.getClass().getName());
```

Comment:
Was there a problem with this? This makes me think `job` can actually be null, but I don't see how that's possible.